drydock_provisioner/control/base.py

```diff
@@ -11,18 +11,36 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import falcon.request as request
 import uuid
 import json
 import logging
 
+from falcon.request import Request
+from falcon import HTTP_403
+from falcon import HTTP_401
+
 import drydock_provisioner.error as errors
 
 class BaseResource(object):
 
-    def __init__(self):
+    def __init__(self, policy_engine=None):
         self.logger = logging.getLogger('control')
```

Comment:
oslo.log is a future enhancement, for now I like having the logger in the class. I don't care for the global variable style.